Gemfile

```diff
@@ -45,7 +45,7 @@ gem "acts_as_tree",                   "~>2.1.0"  # acts_as_tree needs to be requ
 gem "american_date"
 gem "azure-armrest",                  "~>0.1.0"
 gem "color",                          "~>1.8"
-gem "default_value_for",              "~>3.0.1"
+gem "default_value_for",              "~>3.0.1", :git => "git://github.com/matthewd/default_value_for.git", :branch => "rails-50" # https://github.com/FooBarWidget/default_value_for/pull/57
```

Comment:
cool - looks like upstream `default_value_for` is pretty recent and accepting PRs.
I think these are still 4.0 friendly. can we push these?